api/tests/core/external_bookings/cds/test_client.py

```diff
@@ -24,7 +24,10 @@ def create_show_cds(
     shows_tariff_pos_type_ids=None,
     screen_id: int = 50,
     media_id: int = 52,
+    mediaoptions_ids=None,
 ) -> cds_serializers.ShowCDS:
+    if mediaoptions_ids is None:
+        mediaoptions_ids = []
```

Comment:
(détails / style) Pour éviter d'utiliser une liste vide en valeur par défaut (ce qui n'est pas recommandé) et pour éviter d'utiliser `None` (qui ne donne pas vraiment d'indication sur le type attendu), on peut utiliser le tuple vide. Ça indique que l'argument attend une séquence et c'est immutable (donc utilisable comme valeur par défaut). Et on gagne 2 lignes. :)

```suggestion
    mediaoptions_ids=(),
) -> cds_serializers.ShowCDS:
```